subsys/bluetooth/audio/pacs.c

```diff
@@ -226,14 +233,14 @@ static enum bt_audio_context pacs_get_available_contexts_for_conn(struct bt_conn
 	switch (dir) {
 	case BT_AUDIO_DIR_SINK:
 #if defined(CONFIG_BT_PAC_SNK)
-		if (client->snk_available_contexts != NULL) {
+		if (pacs.param.snk_pac && client->snk_available_contexts != NULL) {
```

Comment:
Should `param.snk_pac` be guarded by `CONFIG_BT_PAC_SNK` or should we remove `CONFIG_BT_PAC_SNK` (ditto for src of course)? 

If `CONFIG_BT_PAC_SNK=y` should it be allowed to not register the SNK attributes?

Reply:
> Should param.snk_pac be guarded by CONFIG_BT_PAC_SNK or should we remove CONFIG_BT_PAC_SNK (ditto for src of course)?
I don't think we should remove `CONFIG_BT_PAC_SNK`/CONFIG_BT_PAC_SRC`, these might still be of value for however have no needs of runtime configuration, and want to save memory.

>If CONFIG_BT_PAC_SNK=y should it be allowed to not register the SNK attributes?
I argue that it should be allowed to not register, due to first mentioned reason.

I will add guards though.

Reply:
> It is already guarded though - Im not sure if it was an actual request, or rhetorical.

It is not guarded in the `struct bt_bap_pacs_register_param`. At the time of writing this comment, the application can set `snk_pac=true` when `CONFIG_BT_PAC_SNK=n`. 

We should either guard the fields of the register struct, or remove the Kconfig options